test/Microsoft.ApplicationInsights.AspNet.Tests/RequestTrackingMiddlewareTest.cs

```diff
@@ -8,24 +8,64 @@
     using Microsoft.AspNet.Builder;
     using Microsoft.AspNet.Http.Core;
     using Xunit;
+    using Microsoft.AspNet.Http;
+    using System;
 
     public class RequestTrackingMiddlewareTest
     {
+        private const string HttpRequestScheme = "http";
+        private readonly HostString httpRequestHost = new HostString("testHost");
+        private readonly PathString httpRequestPath = new PathString("/path/path");
+        private readonly QueryString httpRequestQueryString = new QueryString("?query=1");
+
+        private const string ExpectedSdkVersion = "aspnet5";
+
         private ITelemetry sentTelemetry;
 
+        private readonly RequestDelegate nextMiddleware = async httpContext => {
+            httpContext.Response.StatusCode = 200;
+            await httpContext.Response.Body.WriteAsync(new byte[0], 0, 0);
+        };
+
+        private readonly RequestTrackingMiddleware middleware;
+
+        public RequestTrackingMiddlewareTest()
+        {
+            this.middleware = new RequestTrackingMiddleware(
+                this.nextMiddleware,
+                CommonMocks.MockTelemetryClient(telemetry => this.sentTelemetry = telemetry));
+        }
+
         [Fact]
-        public async Task SdkVersionIsPopulatedByMiddleware()
+        public async Task TestSdkVersionIsPopulatedByMiddleware()
```

Comment:
Definitely a nit pick: I tend to find the term `Test` in a test method name wasteful. It is already implied by the `Fact` or `TestMethod` attribute and only takes up space on the editor screen and in test explorer.